app/static/lib/conp-react/src/DatasetElement/index.js

```diff
@@ -48,250 +47,259 @@ const DatasetElement = (props) => {
+                  data-for="downloadTip"
+                />
+                {showDownloadTipText &&
+                  <ReactToolTip id="downloadTip" multiline={true} style={{ Width: "70px", WhiteSpace: "pre-wrap" }}>
+                    Direct download is available for datasets that do not require <br/>
+                    a third-party account to access the data. Datasets requiring a <br/>
+                    third-party account can be downloaded via DataLad.
+                  </ReactToolTip>
+                }
+              </p>
             </div>
           </div>
-          <div className="col col-lg-3 d-flex flex-column justify-content-top align-items-center pr-2">
-            <div className="row align-items-top width-auto">
 
-              <div className="col col-lg-6 d-flex flex-column justify-content-top align-items-center p-2 pr-4">
-                <h7><strong>PROCESS</strong></h7>
-                  {element.cbrain_id ? (
-                    <>
-                      <button className="btn" onClick={openCbrainModal}>
-                        <img
-                          className="cbrain-img"
-                          src="static/img/cbrain-icon-blue.png"
-                          style={{ maxWidth: "60px" }}
-                          alt="Open Dataset"
-                        />
-                      </button>
-                    </>
-                  ) : (
-                    <>
-                      <img
-                          alt="Dataset not available on CBRAIN"
-                          className="cbrain-img justify-content-center align-items-center"
-                          src="static/img/cbrain-icon-grey.png"
-                          style={{ maxWidth: "60px" }}
-                      />
-                    </>
-                  )}
-              </div>
-              <div className="col col-lg-6 d-flex flex-column justify-content-top align-items-center p-2">
-                <h7><strong>DOWNLOAD</strong></h7>
-                <div className="d-flex flex-column">
-                  <button type="button" className="btn btn-outline-secondary m-1" onClick={() => downloadMetadata()}>
-                    Metadata
-                    <div className="spinner-border text-primary" role="status" hidden={!metadataSpinnerState}>
-                      <span className="sr-only">Loading...</span>
-                    </div>
-                  </button>
-                  <div className="alert alert-danger" role="alert" hidden={!metadataErrorState}>
-                    {metadataErrorText}
+          <div className="row align-items-center w-100">
+            <div className="col-10 p-0">
+              <a href={`dataset?id=${element.id}#dataladInstructions`} role="button"
+                   className="btn btn-success m-1">
+                Download With DataLad
+              </a>
+            </div>
+            <div className="col-2 p-2">
+              <p className="card-text pl-1">
```

Comment:
Maybe make this a `div`? React complains for me in the dev console about having a `ReactToolTip` as a descendant of a `p`.